db/migrations/20220220213655_add_unique_constraint_isbn.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
```

Comment:
This is an excellent error handling approach to make sure we only have one copy of each book in favorites